Formula/kn.rb

```diff
@@ -0,0 +1,37 @@
+class Kn < Formula
+  desc "Command-line interface for managing Knative Serving and Eventing resources"
+  homepage "https://github.com/knative/client"
+  url "https://github.com/knative/client.git",
+      tag:      "v0.24.0",
+      revision: "0bbb3ec9d92665d372d8393ef3036ba7e0c069a4"
+  license "Apache-2.0"
+
+  depends_on "bash" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+
+    system "./hack/build.sh", "--fast"
```

Comment:
This is equivalent to:
```
go build -mod=vendor -ldflags "$(build_flags $(basedir))" -o kn ./cmd/...
```
Which, since `build_flags` doesn't exist in the file, is better written like:
```suggestion
    system "go", "build", "-mod=vendor", *std_go_args, "./cmd"
```

Reply:
thanks for the suggestions. I've updated the formula